### PR TITLE
Convert java files to kotlin

### PR DESCRIPTION
**Pull request by Astach**
That's not all of them, but I don't want to push a too big PR

### CODE REVIEW

File: ui/src/main/java/com/schibsted/account/ui/login/screen/identification/IdentificationPresenter.kt [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+class IdentificationPresenter(
+        private val identificationView: IdentificationContract.View,
+        private val provider: InputProvider<Identifier>?,
+        private val flowSelectionListener: FlowSelectionListener?) : IdentificationContract.Presenter {
+
+    init {
+        identificationView.setPresenter(this)
+    }
+
+    private fun identifyUser(identifierType: Identifier.IdentifierType, input: String?, identifier: InputField) {
```
Comment (Nexcius) [inline ui/src/main/java/com/schibsted/account/ui/login/screen/identification/IdentificationPresenter.kt]: `input` should not be optional

File: ui/src/main/java/com/schibsted/account/ui/login/screen/identification/IdentificationPresenter.kt [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                        identificationView.showError(identifier)
+                    }
+                    identificationView.hideProgress()
+                    trackError(error)
+                }
+            }
+        })
+    }
+
+    private fun getAccountStatus(identifierType: IdentifierType, input: String?, identifier: InputField, allowSignUp: Boolean, signUpErrorMessage: String?) {
```
Comment (Nexcius) [inline ui/src/main/java/com/schibsted/account/ui/login/screen/identification/IdentificationPresenter.kt]: `input` not optional

File: ui/src/main/java/com/schibsted/account/ui/login/screen/identification/ui/AbstractIdentificationFragment.kt [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val args = savedInstanceState ?: arguments
+        args?.let {
+            if (args.getParcelable<Parcelable>(KEY_UI_CONF) != null) {
+                uiConf = args.get(KEY_UI_CONF) as UiConfiguration
+            }
+        }
+
+        if (!::uiConf.isInitialized && context != null) {
```
Comment (Nexcius) [inline ui/src/main/java/com/schibsted/account/ui/login/screen/identification/ui/AbstractIdentificationFragment.kt]: `::` not neeed

Comment (Astach) [reply ui/src/main/java/com/schibsted/account/ui/login/screen/identification/ui/AbstractIdentificationFragment.kt]: how do you access `isInitialized` then?

**Pull request merged**